paddle/framework/scope.cc

```diff
@@ -62,5 +64,17 @@ void Scope::DropKids() {
   kids_.clear();
 }
 
+std::once_flag feed_variable_flag;
+
+framework::Scope* GetGlobalScope() {
+  static std::unique_ptr<framework::Scope> g_scope{nullptr};
+  std::call_once(feed_variable_flag, [&]() {
+    g_scope.reset(new framework::Scope());
+    g_scope->NewVar("feed_value");
```

Comment:
use const:
```cpp
const std::string kFeedValue = "feed_value";
const std::string kFetchValue = "fetch_value";
```

Reply:
why do we use a global scope and var to store the feed value?

Reply:
there is a detailed discussion on this: https://github.com/PaddlePaddle/Paddle/pull/4599.
Any suggestion on an alternative implementation is welcomed.